[last medical report says](https://drive.google.com/file/d/1PlqmtNuURQM8JYrTedqvlBcssA-gsLLX/view) (p. 27) says that

In [1]:
height = 176.4
weight = 78.7
bmi = 25.3

this agrees with formula at https://www.cdc.gov/healthyweight/assessing/bmi/childrens_bmi/childrens_bmi_formula.html

In [4]:
def compute_bmi(h,w): return w/(h/100)**2
compute_bmi(height,weight)

25.291673736766057

inversing, to have a BMI<=24.9 (which is "healthy"), I need to have weight <= than

In [6]:
import numpy as np
# see https://drive.google.com/file/d/1PlqmtNuURQM8JYrTedqvlBcssA-gsLLX/view, p.27
HEALTHY_BMI = 24.9
f"{max([w for w in np.arange(70,80,0.01) if compute_bmi(height,w)<=HEALTHY_BMI]):.2f}"

'77.48'

==> I should try to maintain my weight in range 76.5~77.5